kafka-admin/src/main/java/io/strimzi/admin/kafka/admin/handlers/CommonHandler.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.admin.kafka.admin.handlers;
+
+import graphql.schema.DataFetchingEnvironment;
+import io.strimzi.admin.kafka.admin.AdminClientWrapper;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.ext.web.RoutingContext;
+import org.apache.kafka.common.config.SaslConfigs;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Map;
+
+public class CommonHandler {
+    protected static final Logger log = LogManager.getLogger(CommonHandler.class);
+
+    protected static void adjustToken(Map acConfig, DataFetchingEnvironment env) {
+        RoutingContext rc = env.getContext();
+        String token = rc.request().getHeader("Authorization");
+        if (token != null) {
+            if (token.startsWith("Bearer ")) {
+                token = token.substring("Bearer ".length());
+            }
+            log.info("auth token is {}", token);
```

Comment:
I thought logging the token was a security breach. Does showing the token itself provide anything of value?